docs/development/design/privacy-levels.rst

```diff
@@ -0,0 +1,93 @@
+Privacy Levels
+==============
+
+This document describes how to handle and unify privacy levels
+on the community and commercial version of Read the Docs.
+
+Current state
+-------------
+
+Currently, we have three privacy levels for projects and versions:
+
+#. Public
+#. Private
+#. Protected (currently hidden)
+
+These levels of privacy aren't clear and bring confusion to our users.
+Also, the private level doesn't makes sense on the community site,
+since we only support public repos.
+
+Places where we use the privacy levels are:
+
+- On serving docs
+- Footer
+- Dashboard
+
+Project level privacy
+---------------------
+
+Project level privacy was meant to control the dashboard visibility.
+
+This privacy level brings to confusion when users want to make a version public.
+We should remove all the project privacy levels.
+
+For the community site the dashboard would be always visible,
+and for the commercial site, the dashboard would be always hidden.
+
+Some other ideas about keeping the privacy level is to dictate the default version level of new versions.
+This can be (or is going to be) possible with automation rules.
+
+Version level privacy
+---------------------
+
+Version level privacy is mainly used to restrict access to documentation.
+For public level, everyone can access to the documentation.
+For private level, only users that are maintainers or that belong to a team with access
+(for the commercial site)
+can access to the documentation.
+
+The protected privacy level was meant to hid versions from listings and search.
+For the community site these versions are treated like public versions,
+and on the commercial site they are treated like private.
+
+The protected privacy level is currently hidden.
+To keep the behavior of hid the versions from listings and search,
+a new field should be added to the Version model: ``hidden``.
+The privacy level (public or private) would be respected to determinate access to the documentation.
+
+For the community site, the privacy level would be public and can't be changed.
+
+The default privacy level of new versions for the commercial site would be ``private``.
```

Comment:
This is taking into consideration that we drop completely the project privacy level.

Reply:
Not the project default privacy level though, only the project privacy level is being dropped as I understand our needs here.

Reply:
I was thinking in just dropping the field completely. If we want the users to have a default privacy level, they could use automation rules in the future. For .org all versions are public, so that isn't needed there. And for .com, well, most users want private docs most of the time. I think leaving it as default privacy level will lead to the same confusion users have right now. But I'm not sure how that field would be replaced.